src/olympia/blocklist/tests/test_commands.py

```diff
@@ -64,11 +71,21 @@ def test_regex(self):
             assert block.kinto_id == '*' + this_block['id']
             assert block.include_in_legacy
             assert block.modified == datetime(2019, 11, 29, 22, 22, 46, 785000)
+        assert KintoImport.objects.count() == 6
+        assert KintoImport.objects.filter(
+            outcome=KintoImport.OUTCOME_NOMATCH).count() == 4
+        assert KintoImport.objects.filter(
+            outcome=KintoImport.OUTCOME_NOTFIREFOX).count() == 1
+        kim = KintoImport.objects.get(
```

Comment:
Nit: In `src/olympia/blocklist/tasks.py` there's a variable called `kin`, and here there's one called `kim`, which both seem to refer to a `KintoImport` object.

Reply:
https://github.com/mozilla/addons-server/pull/13274/commits/05b0c2536b4bf1b1f310a482ad84345acf39477d